Documentation/DCR/ContainerLogV2Extension.json

```diff
@@ -0,0 +1,40 @@
+{
+    "location": "<la-workspace-region>",
+    "properties": {
+        "dataSources": {
+            "extensions": [
+                {
+                    "streams": [
+                        "Microsoft-ContainerLogV2-HighScale"
+                    ],
+                    "extensionName": "ContainerLogV2Extension",
+                    "extensionSettings": {
+                        "dataCollectionSettings": {
+                            "namespaces": ["app-team-k8s-namespace-1", "app-team-k8s-namespace-2"]
```

Comment:
is there a scenario to include all namespaces?

Reply:
> is there a scenario to include all namespaces?

This is just sample for documentation purpose.